server/app/src/test/java/org/apache/james/app/spring/JamesSpringContextTest.java

```diff
@@ -50,6 +50,6 @@ public void springShouldLoadAndAddOnlyOneQuotaUpdaterListener() {
         InVMEventBus eventBus = context.getBean(InVMEventBus.class);
 
         assertThat(eventBus.registeredGroups())
-            .containsExactlyInAnyOrder(ListeningCurrentQuotaUpdater.GROUP, LuceneMessageSearchIndex.GROUP);
+            .containsExactlyInAnyOrder(MailboxAnnotationListener.GROUP, LuceneMessageSearchIndex.GROUP);
```

Comment:
how is it related?

Reply:
This is outdated.

Shortly: main configuration files do have more mailbox listeners than the ones previously used for testing.